conda_content_trust/cli.py

```diff
@@ -392,7 +392,7 @@ def fn_addsig():
                 conda_content_trust.root_signing.sign_root_metadata_dict_via_gpg(
                     metadata, key
                 )
-            except:
+            except:  # noqa: E722
```

Comment:
What should this one catch?

Reply:
the entire codebase relies to much on general exceptions without a testcase making clear why those exist.... remove requires looking deeper into it.

Reply:
Thanks, IMO it is better not to silence this error. Or fix it by listing exceptions (even `except Exception:` is better than bare `except:`), and I see you've found a short list to add.